distributor-widget/reference.md

```diff
@@ -104,6 +104,14 @@ distributor.setEndDate("2019-12-18")
 
 Sets a new voucher code value.
 
+#### disableTracking\(\)
```

Comment:
I think you can do it just like this without skipping it.
```suggestion
#### disableTracking()
```

Reply:
Oh I didn't know, will fix.